tests/lib/config/config-validator.js

```diff
@@ -103,6 +120,12 @@ describe("Validator", () => {
             assert.doesNotThrow(fn);
         });
 
+        it("should do nothing with an invalid config when severity is off", () => {
+            const fn = validator.validate.bind(null, { rules: { "mock-required-options-rule": "off" } }, "tests");
+
+            assert.doesNotThrow(fn);
+        });
```

Comment:
Just to be absolutely sure, can we have a test with `["off"]` as the configured set of options? If nothing else, it could prevent a regression later.